libretroshare/src/util/rsdir.cc

```diff
@@ -840,7 +840,7 @@ int RsDirUtil::createLockFile(const std::string& lock_file_path, rs_lock_handle_
 	/******************************** WINDOWS/UNIX SPECIFIC PART ******************/
 }
 
-void RsDirUtil::releaseLockFile(rs_lock_handle_t lockHandle)
+void RsDirUtil::releaseLockFile(rs_lock_handle_t &lockHandle)
```

Comment:
missing const

Reply:
I can't as lockHandle is assigned a value after...

Reply:
well, then dont change it at all! Without the const, it would modified the variable passed as parameter. This particular case is a perfect example of why the const is useful when doing such changes!